docker-compose.prod.yml

```diff
@@ -27,14 +27,19 @@ services:
 
   webapp:
     container_name: bigcapital-webapp
-    image: ghcr.io/bigcapitalhq/webapp:latest
+    build:
+      context: ./
+      dockerfile: packages/webapp/Dockerfile
```

Comment:
Webapp container building is done by CI/CD (Github actions) by `Dockerfile` that under `/packages/webapp`, when configure them in prod we're just pulling out from public registry.

Reply:
Yes, that absolutely should be the way to do it. I was tinkering with the code (to fix invoicing) and needed to change the source.